integration_tests/e2e/about.cy.ts

```diff
@@ -158,3 +151,29 @@ describe('Rendering', () => {
     cy.contains('.govuk-accordion__section', 'Health and wellbeing').find('.govuk-body').should('have.length', 2)
   })
 })
+
+describe('Rendering READ_ONLY', () => {
+  beforeEach(() => {
+    cy.createSentencePlan().then(planDetails => {
+      cy.wrap(planDetails).as('plan')
+      cy.openSentencePlan(planDetails.oasysAssessmentPk, AccessMode.READ_ONLY)
+      cy.get('.moj-primary-navigation__container').contains('a', 'About').click()
+    })
+  })
+
+  it('Should check the page rendered correctly with no Create Goal button', () => {
+    cy.url().should('include', '/about')
```

Comment:
Probably a throwaway thought, but I never know how to feel about verifying the URL is correct at the start of test if the URL was navigated to in the beforeEach step - why not just verify in the beforeEach?

Reply:
Yes, this has probably been propagated via copy/paste over time. I'll take this out from here.

Reply:
Fixed in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/322/commits/3dfc627af170843f68f4b609ff6f0d4803478718